lib/screens/search/search.dart

```diff
@@ -55,5 +82,7 @@ class _SearchScreenState extends State<SearchScreen> {
         ),
       ),
     );
+    queryParam = null;
```

Comment:
you shouldn't set state directly from build();

Reply:
what are you trying to do here?

Reply:
Updated how this is handled.